src/components/dialogs/parameters/load-flow-parameters.tsx

```diff
@@ -380,16 +442,24 @@ const SpecificLoadFlowParameters = ({
             infoText={subText}
         >
             <FlatParameters
-                sx={styles.parameterName}
```

Comment:
I think you will loose the bold style here ?

Reply:
I would agree, if sx was actually used in FlatParameters
It's not in the interface, and it's not forwarded to the component either